### DISCUSSION TITLE
Playing automatically vertical videos on 2nd monitor

### DISCUSSION THREAD

**Discussion by Qliphort8** [mpv-player/mpv#14456] [Q&A] [answered]
right now i have these lines in my config:
```
[portrait on 2nd monitor]
profile-cond=height >= 1081
fs-screen = 2
[landscape on 1st monitor]
profile-cond=width >= 1280
fs-screen = 0
```

this way vertical videos are automatically started on my vertical monitor and landscape videos on my main monitor.

Problem is when the video is too low or too high of a resolution it mixes it up so my question is if there is a better way to do that? I couldn't find a way to detect the aspect ratio

**Comment by guidocella** [comment, marked as answer]
`height > width`

**Reply by Qliphort8** [reply]
thank you so much
